deployer/run.sh

```diff
@@ -169,9 +169,13 @@ keytool -noprompt -import -v -trustcacerts -alias $hawkular_cassandra_alias -fil
 
 echo "Importing the CA Certificate into the Cassandra Truststore"
 keytool -noprompt -import -v -trustcacerts -alias ca -file ${dir}/ca.crt -keystore $dir/hawkular-cassandra.truststore -trustcacerts -storepass $hawkular_cassandra_truststore_password
+keytool -noprompt -import -v -trustcacerts -alias metricca -file ${dir}/hawkular-metrics-ca.cert -keystore $dir/hawkular-cassandra.truststore -trustcacerts -storepass $hawkular_cassandra_truststore_password
+keytool -noprompt -import -v -trustcacerts -alias cassandraca -file ${dir}/hawkular-cassandra-ca.cert -keystore $dir/hawkular-cassandra.truststore -trustcacerts -storepass $hawkular_cassandra_truststore_password
 
 echo "Importing the CA Certificate into the Hawkular Metrics Truststore"
 keytool -noprompt -import -v -trustcacerts -alias ca -file ${dir}/ca.crt -keystore $dir/hawkular-metrics.truststore -trustcacerts -storepass $hawkular_metrics_truststore_password
+keytool -noprompt -import -v -trustcacerts -alias metricsca -file ${dir}/hawkular-metrics-ca.cert -keystore $dir/hawkular-metrics.truststore -trustcacerts -storepass $hawkular_metrics_truststore_password
```

Comment:
I have them all set as an optional secret for the deployer and the deployer is the only component meant to use that secret.

Whether the cert is generated of comes from the secret, it will end up being copied to ${dir}, so its in the same location regardless.